src/lib/api/api.go

```diff
@@ -307,7 +307,10 @@ func (a *API) QuickTestInput(w http.ResponseWriter, r *http.Request) {
 	case "POST":
 		go req.Send(request.POST, response)
 	default:
-		fmt.Printf("invalid request method: %s\n", t.Method)
+		go func() {
+			fmt.Printf("invalid request method: %s\n", t.Method)
+			response <- request.ResponseWrapper{ReponseStringified: "Invalid method", Status: 404}
```

Comment:
- "When is the default method being called?" Might as well comment that above this.
- Should we call the `ResponseWrapper` on POST requests as well? What makes the default case so special?

Reply:
In case of POST and GET request. We are calling req.Send() in this function we are already passing Response Wrapper Object.
Default method will be called when the payload string with key "method " will have any other value apart from "GET" and "POST"

Thanks

Reply:
@aquibbaig can I add comment this way:
default: //Default will be called when the payload string with key "method " will have any othe...
go func() {
			fmt.Printf("invalid request method: %s\n", t.Method)
			response <- request.ResponseWrapper{ReponseStringified: "Invalid method", Status: 404
}

Reply:
@Sanketwable Looks good to me, lets wait for another set of reviews.